model/trace/http.yaml

```diff
@@ -66,6 +66,7 @@ groups:
         requirement_level: opt_in
         brief: Local socket port. Useful in case of a multi-port host.
       - ref: client.address
+        sampling_relevant: true
```

Comment:
is there a use case where client-address is useful for sampling? would someone sample out spans from one IP but not another? Like sampling out internal traffic or blocking specific IPs for DDoS protection?

Reply:
I was thinking e.g. for prioritizing requests from a specific address block (company / country) or even a specific IP (e.g. while debugging)